Replace subjective "MUST be small" with guidelines (#602)
publiccodenet/standard: criteria/continuous-integration.md

* All functionality in the source code MUST have automated tests.
* Contributions MUST pass all automated tests before they are admitted into the codebase.

> * The codebase MUST have guidelines which state how contributions are expected to be organized.

Author: Yes, indeed, as "should" could be perceived as "SHOULD", finding alternative phrasing seems best.
Open to other alternatives.

> * The codebase MUST have guidelines which state how contributions are expected to be organized.

Reviewer: What about:
```suggestion
* The codebase MUST have guidelines explaining how to organize contributions.
```

## Management: what you need to do

* Make sure to test with real end users as quickly and often as possible.

> * Plan the work to deliver small parts very often instead of large parts less frequently.

Reviewer: I love this as a solution to @bvhme's comment.
